src/index.es.ts

```diff
@@ -10,18 +9,71 @@ import {
   getInsightsAnonymousUserToken,
 } from './helpers';
 import { createInfiniteHitsSessionStorageCache } from './lib/infiniteHitsCache';
+import { deprecate } from './lib/utils';
 
-const instantsearch = (options: InstantSearchOptions): InstantSearch =>
+type InstantSearchModule = {
+  (options: InstantSearchOptions): InstantSearch;
+  version: string;
+
+  // @major remove these in favour of the exports
+  /** @deprecated */
```

Comment:
@francoischalifour suggested *not* deprecating these yet, as we aren't sure where the export should best be from. At the moment it's /es/helpers, but that could change? 

Personally I'm more of the opinion to deprecate them with /es/helpers as a workaround, still leaving the option open for a more thorough review of what's exported where in a major version

wdyt @tkrugg / @shortcuts

Reply:
As long as we have clear warnings etc., I don't see any issue deprecating them